packages/page-accounts/src/Accounts/index.tsx

```diff
@@ -106,7 +106,8 @@ function Overview ({ className = '', onStatusChange }: Props): React.ReactElemen
   }, [accountsWithInfo]);
 
   const header = useRef([
-    [t('accounts'), 'start', 3],
+    [], [],
```

Comment:
```suggestion
    [t('favorite')], [],
```
Table with header looks really odd when there's no name for the first column.

Reply:
✅